Assets/_PlatformerDevelopment/Scripts/Enemy/LongRangeEnemy.cs

```diff
@@ -0,0 +1,32 @@
+﻿using UnityEngine;
+
+namespace PersonalDevelopment
+{
+    public class LongRangeEnemy : Enemy
```

Comment:
Should rename this to Long Range Attack. 

What we potentially want, is to have the game object attacha bunch of "actions"
So I didn't name my script/component Melee Enemy, I named it movement. So that script is attached to anyone who wants to move. 

This is more personal preference. 
Should be one component that  setups/initializes all the action scripts - if they exist on the gameobject.